server/manager/src/main/java/org/apache/accumulo/manager/recovery/RecoveryManager.java

```diff
@@ -198,11 +198,9 @@ public boolean recoverLogs(KeyExtent extent, Collection<Collection<String>> walo
         synchronized (this) {
           if (!closeTasksQueued.contains(sortId) && !sortsQueued.contains(sortId)) {
             AccumuloConfiguration aconf = manager.getConfiguration();
-            @SuppressWarnings("deprecation")
             LogCloser closer = Property.createInstanceFromPropertyName(aconf,
-                aconf.resolve(Property.MANAGER_WAL_CLOSER_IMPLEMENTATION,
-                    Property.MANAGER_WALOG_CLOSER_IMPLEMETATION),
-                LogCloser.class, new HadoopLogCloser());
+                aconf.resolve(Property.MANAGER_WAL_CLOSER_IMPLEMENTATION), LogCloser.class,
```

Comment:
another resolve on single property can be simplified; there's a bunch more, but I'll stop mentioning it.